build.sbt

```diff
@@ -76,16 +76,31 @@ lazy val publishSettings : Seq[Setting[_]] = Seq(
 // should not be set directly. It is the same as the Maven version and derived automatically from `baseVersion` and
 // `baseVersionSuffix`.
 globalVersionSettings
-baseVersion in Global := "2.13.0"
-baseVersionSuffix in Global := "SNAPSHOT"
+Global / baseVersion       := "2.13.0"
+Global / baseVersionSuffix := "SNAPSHOT"
+ThisBuild / organization   := "org.scala-lang"
+ThisBuild / homepage       := Some(url("https://www.scala-lang.org"))
+ThisBuild / startYear      := Some(2002)
+ThisBuild / licenses       += (("Apache-2.0", url("https://github.com/scala/scala/blob/2.13.x/LICENSE")))
+ThisBuild / headerLicense  := Some(HeaderLicense.Custom(
+  s"""                     __
+     |     ________ ___   / /  ___     Scala
+     |    / __/ __// _ | / /  / _ |    (c) EPFL
+     |  __\\ \\/ /__/ __ |/ /__/ __ |    (c) Lightbend, Inc.
+     | /____/\\___/_/ |_/____/_/ | |    ${(ThisBuild/homepage).value.get}
+     |                          |/     Licensed under Apache License 2.0
```

Comment:
I wouldn't mind having this fine piece of art printed, framed, put on a shelf and move on to a future of lighter headers in scala/scala source files 😊

Reply:
haha I'm so used to it I didn't even occur to me to suggest a change

agree, it isn't worth the vertical real estate.

Reply:
It only looks ugly here because I needed to escape backslash to `\\` even though I am using tripplequote `"""`. Is that an expected behavior with `s"""`?

Reply:
May I interest you in Bloody theme during the month of October?

```
  ██████  ▄████▄   ▄▄▄       ██▓    ▄▄▄      
▒██    ▒ ▒██▀ ▀█  ▒████▄    ▓██▒   ▒████▄    
░ ▓██▄   ▒▓█    ▄ ▒██  ▀█▄  ▒██░   ▒██  ▀█▄  
  ▒   ██▒▒▓▓▄ ▄██▒░██▄▄▄▄██ ▒██░   ░██▄▄▄▄██ 
▒██████▒▒▒ ▓███▀ ░ ▓█   ▓██▒░██████▒▓█   ▓██▒
▒ ▒▓▒ ▒ ░░ ░▒ ▒  ░ ▒▒   ▓▒█░░ ▒░▓  ░▒▒   ▓▒█░
░ ░▒  ░ ░  ░  ▒     ▒   ▒▒ ░░ ░ ▒  ░ ▒   ▒▒ ░
░  ░  ░  ░          ░   ▒     ░ ░    ░   ▒   
      ░  ░ ░            ░  ░    ░  ░     ░  ░
         ░                                   
```

Reply:
Careful not to trigger font bikeshedding 😱

Reply:
I took the liberty to replace "NSC -- new Scala compiler" with just "Scala". Are you folks cool with that?